plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -2030,6 +2039,78 @@ public void testIncorrectIcebergFileSizes()
         dropTable("test_iceberg_file_size");
     }
 
+    @Test
+    @Flaky(issue = "https://github.com/trinodb/trino/issues/5172", match = "Couldn't find operator summary, probably due to query statistic collection error")
+    public void testSplitPruningForFilterOnPartitionColumn()
+    {
+        String tableName = "nation_partitioned_pruning";
+
+        assertUpdate("DROP TABLE IF EXISTS " + tableName);
+
+        // disable writes redistribution to have predicatable number of files written per partition (one).
```

Comment:
did you mean `predicatable` or `predictable`?

Reply:
predictable